src/lib/loaders/loaders_with_authentication/vimeo.ts

```diff
@@ -0,0 +1,22 @@
+import factories from "../api"
+import { accessTokenByRoleLoader } from "./utils/accessTokenByRoleLoader"
+
+export const vimeoLoaders = (accessToken, opts) => {
+  const { vimeoLoaderWithAuthenticationFactory } = factories(opts)
+
+  const { accessTokenLoader } = accessTokenByRoleLoader(
+    // TODO: What role do partners updating artworks have?
+    "team",
```

Comment:
@jonallured - not sure. Do you know?

Reply:
I think that they don't have a special role, they have access to do things on PartnerArtwork records that they create but I'd have to do more research to be sure.

Reply:
Shit which reminds me - will we be adding the Vimeo fields to `Artwork` records or `PartnerArtwork` records...